src/main/webapp/i18n/de/metis.json

```
@@ -104,6 +104,11 @@
             "answerPost": {
                 "created": "Antwort erfolgreich erstellt",
                 "deleted": "Antwort erfolgreich gelöscht"
+            },
+            "userAuthorityTooltips": {
+                "instructor": "Dozent: Verantwortlich für die Kursdurchführung.",
```

```suggestion
                "instructor": "Lehrkraft: Verantwortlich für die Kursdurchführung.",
```
If I remember the guidelines correctly.

---

thx